python/setup.py

```diff
@@ -274,6 +276,77 @@ def download_pickle5(pickle5_dir):
                 wzf.close()
 
 
+def patch_isdir():
+    """
+    Python on Windows is having hard times at telling if a symlink is
+    a directory - it can "guess" wrong at times, which bites when
+    finding packages. Replace with a fixed version which unwraps links first.
+    """
+    orig_isdir = os.path.isdir
+
+    def fixed_isdir(path, debug=False):
+        while os.path.islink(path):
+            try:
+                link = os.readlink(path)
+            except OSError:
+                break
+            path = os.path.abspath(os.path.join(os.path.dirname(path), link))
+        return orig_isdir(path)
+
+    os.path.isdir = fixed_isdir
+
+
+def replace_symlinks_with_junctions():
+    _LINKS = {
```

Comment:
this should probably have a comment telling to update the list if a new symlink is introduced in the source tree...